build.gradle

```diff
@@ -63,35 +64,36 @@ license {
 }
 
 tasks.named('wrapper', Wrapper).configure {
-    gradleVersion = '8.4'
+    gradleVersion = '8.10.1'
```

Comment:
you also didnt run the gradle wrapper task as there are updates to the shell scripts

Reply:
That is true, forgot about that!

Reply:
Rolled back Gradle, consider fixed